### ISSUE TITLE
How to start jackd from the python module?

### ISSUE THREAD

**Issue by johnstile** [spatialaudio/jackclient-python#55]
Creating an instance of jack.Client will spawn an instance of jackd server, if it is not already running.
Is there a way to specify the options to the spawned jackd server from within this python package?

**Comment by mgeier** [comment]
No, the only option is whether to start it automatically or not. This can be controlled with `no_start_server`.

But JACK itself has some configuration files which you can use to control those options.
I don't know exactly what files those are, but if you for example use `qjackctl` to specify the JACK settings, those will also be used if you start `jackd` from this Python module.
